draft-js-side-toolbar-plugin/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+### 2.0.6
```

Comment:
This is a breaking change, so let's bump to `3.0.0`

Reply:
Modified!